recipe/reader_test.go

```diff
@@ -221,6 +222,15 @@ func TestReaderRead(t *testing.T) {
 			compareRecipes(t, expected[i], r)
 		}
 	})
+
+	t.Run("should return error if version is missing/incorrect", func(t *testing.T) {
+		reader := recipe.NewReader()
+		_, err := reader.Read("./testdata/missing-version.yaml")
```

Comment:
@StewartJingga I think the goal is that we want to have all recipes to have this so that we can support upgrades better in future. We can add it but I am thinking since lint will inform it and this is just an additive change and not a change in any configuration. Should it be fine? what do you think? We can mark it breaking in release.

Reply:
We should definitely mark it as breaking. Will add `!` to the PR name.

Ideally maybe we can start with notice (similar to deprecation notice), but for now I think we can still afford breaking changes as long as we inform all of current meteor users. We are also locking our version so it won't break our deployment.